ajax/libs/jimp/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "jimp",
+  "filename": "jimp.min.js",
+  "version": "0.2.27",
+  "description": "An image processing library written entirely in JavaScript (i.e. zero external or native dependencies).",
+  "keywords": [
+    "image",
+    "image processing",
+    "image manipulation",
+    "png",
+    "jpg",
+    "jpeg",
+    "bmp",
+    "resize",
+    "scale",
+    "crop"
+  ],
+  "author": "Oliver Moran <oliver.moran@gmail.com>",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/oliver-moran/jimp.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/oliver-moran/jimp.git",
+    "fileMap": [
+      {
+        "basePath": "",
```

Comment:
The `basePath` is `browser/lib`.
Because `basePath` mean that which files under this folder should cdnjs automatically host.
ref: https://github.com/cdnjs/cdnjs/blob/master/documents/autoupdate.md